src/components/ScreenWrapper/index.js

```diff
@@ -108,9 +117,13 @@ class ScreenWrapper extends React.Component {
                                 styles.flex1,
                                 paddingStyle,
                             ]}
+                            // eslint-disable-next-line react/jsx-props-no-spreading
+                            {...(this.props.environment === CONST.ENVIRONMENT.DEV) ? this.panResponder.panHandlers : {}}
```

Comment:
Could you please move the parenthesis to make this a bit more clear?
```javascript
{...(this.props.environment === CONST.ENVIRONMENT.DEV ? this.panResponder.panHandlers : {})}
```

Reply:
Please message me on NewDot when it's updated and I'll approve.

Reply:
@neil-marcellini okay